main.py

```diff
@@ -1,10 +1,49 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
+JSON_FILE = "players.json"
+
+
 def main() -> None:
-    pass
+    with open(JSON_FILE) as f:
+        data = json.load(f)
+    for player, info in data.items():
+        if not Race.objects.filter(name=info["race"]["name"]).exists():
+            Race.objects.create(
+                name=info["race"]["name"],
+                description=info["race"]["description"],
+            )
+
+        race = Race.objects.get(name=info["race"]["name"])
```

Comment:
This will not affect the number of database queries but may improve readability. For this, use the `get_or_create() `method. Just try it )

Reply:
The same in other similar cases